backend/api/EventHandlers/MissionEventHandler.cs

```diff
@@ -89,7 +84,14 @@ private async void OnMissionRunCreated(object? sender, MissionRunCreatedEventArg
 
         private async void OnRobotAvailable(object? sender, RobotAvailableEventArgs e)
         {
-            if (e.Robot.Status != RobotStatus.Available)
+            if (
+                !(
+                    e.Robot.Status
+                    is RobotStatus.Available
+                        or RobotStatus.Docked
+                        or RobotStatus.ReturningHome
```

Comment:
Consider putting these three in a variable `var RobotStatusesWhereRobotCanStartMission` (or something similar) for better readabillity